Samsung/A32-SystemUI/res/values/bools.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
none of those bools are in sysui

Reply:
i dont understand. does that mean my whole system ui overlay is useless?